apps/bundle-size/src/utils/prepareFixture.js

```diff
@@ -0,0 +1,90 @@
+// @ts-check
+
+const { default: Ajv } = require('ajv');
+const Babel = require('@babel/core');
+const fs = require('fs-extra');
+const path = require('path');
+
+const ajv = new Ajv();
+
+/**
+ * A schema to validate metadata for bundle size stories.
+ */
+const schema = {
```

Comment:
**q:** can we use standard json-schema -> defined within `schema.json` ?

Reply:
Yes, I will extract it into `.json` file 👍

Reply:
Done in 0843f0b.